src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs

```diff
@@ -144,10 +144,9 @@ public Task ShutdownAsync(CancellationToken _)
         [JsonRpcMethod(Methods.ExitName)]
         public Task ExitAsync(CancellationToken _)
         {
-            Contract.ThrowIfFalse(_shuttingDown, "Shutdown has not been called yet.");
-
             try
             {
+                ShutdownRequestQueue();
```

Comment:
Remove the contract and added another call to shutdown the queue.  multiple requests to shutdown the queue should no-op.

the client currently does not call shutdown before exit when full closing VS.  So in order to capture telemetry when the server shuts down, I need to ensure the logger dispose is called here.